graylog2-server/src/main/java/org/graylog/events/processor/aggregation/PivotAggregationSearch.java

```diff
@@ -144,9 +149,18 @@ public AggregationResult doSearch() throws EventProcessorException {
             });
 
             // If we have only EmptyParameterErrors, just return an empty Result
-            if (! (errors.stream().filter(e -> !(e instanceof EmptyParameterError)).count() > 1)) {
+            if (errors.stream().allMatch(e -> e instanceof EmptyParameterError)) {
```

Comment:
we might consider backporting this fix by itself

Reply:
I'll check with triage